tests/plugins/stylescombo/listreuse.js

```diff
@@ -75,10 +88,14 @@
 			bot.setHtmlWithSelection( '<ul class="ul-a"><li><ul><li>^</li></ul></li></ul>' );
 
 			stylesCombo.createPanel( editor );
-			stylesCombo.onOpen();
+			stylesCombo.onOpen( editor );
 
 			assert.areEqual( '', stylesCombo._.value );
-			assert.isFalse(stylesCombo._.list.isMarked('ul-a'))
+			assert.isFalse(stylesCombo._.list.isMarked('ul-a'));
+
+			stylesCombo.onClick( 'ul-a' );
+			assert.areSame( '<ul class="ul-a"><li><ul class="ul-a"><li>&nbsp;</li></ul></li></ul>', bot.getData( true ) );
```

Comment:
nbsp is added so that the current empty list item (denoted by caret position ^ originally) doesn't vanish when calling getData